docs/source/libraries/sklearn.rst

```diff
@@ -0,0 +1,225 @@
+scikit-learn
+============
+
+ELI5 supports many estimators, transformers and other components
+from scikit-learn_ library.
+
+.. _scikit-learn: https://github.com/scikit-learn/scikit-learn
+
+Additional ``explain_weights`` and ``explain_prediction`` parameters
+--------------------------------------------------------------------
+
+For all scikit-learn classifiers and regressors :func:`eli5.explain_weights`
+and :func:`eli5.explain_prediction` support additional keyword arguments.
+Additional :func:`eli5.explain_weights` parameters:
+
+* ``vec`` is a vectorizer instance used to transform
+  raw features to the input of the classifier or regressor
+  (e.g. a fitted CountVectorizer instance); you can pass it
+  instead of ``feature_names``.
+
+Additional :func:`eli5.explain_prediction` parameters:
+
+* ``vec`` is a vectorizer instance used to transform
+  raw features to the input of the classifier or regressor
+  (e.g. a fitted CountVectorizer instance); you can pass it
+  instead of ``feature_names``.
+
+* ``vectorized`` is a flag which tells eli5 if ``doc`` should be
+  passed through ``vec`` or not. By default it is False, meaning that
+  if ``vec`` is not None, ``vec.transform([doc])`` is passed to the
+  estimator. Set it to False if you're using ``vec`` to get feature names,
```

Comment:
I would say "if you are passing ``vec`` instead of ``feature_names``", or just "if you are passing ``vec``" - to make it clear that what matters is what you pass, not how you get the feature names.